<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>LIBRARY MANAGEMENT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPush">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Push</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPop">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Pop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPeek">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Peek</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonEnqueue">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Enqueue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDequeue">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>120</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Dequeue</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBook">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>BOOK:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RESERVATION:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditReservation">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textOutput">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>221</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPeekReservation">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>120</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Peek Reservation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonState">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Display State</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
